cmd/meroxa/root/apps/init.go

```diff
@@ -95,10 +96,14 @@ func validateAppName(name string) (string, error) {
 }
 
 func (i *Init) Execute(ctx context.Context) error {
```

Comment:
Adding ruby caused a linter error:`cyclomatic complexity 16 of func`. So I simplified the function logic